app/controllers/api/v1/reviews_controller.rb

```diff
@@ -0,0 +1,20 @@
+class Api::V1::ReviewsController < ApplicationController
+
+  def create
+    review = Review.new(review_params)
+    coffee_shop = CoffeeShop.find(params[:coffee_shop_id])
+    review.coffee_shop = coffee_shop
```

Comment:
You might consider updating your reviews migration so that only a User can create a review rather than any visitor on your site

Reply:
We have this being worked on in another branch!

Reply:
sounds good! 👌